### ISSUE TITLE
[Bug] App won't start - System.FormatException 

### ISSUE THREAD

**Issue by swoy45** [t1m0thyj/WinDynamicDesktop#487]
- WDD Version: 5.1.0
- OS Version: Win11 Build 22621.608

Steps to Reproduce:

1. Set regional format in system to Japanese
2. Set AM/PM marks in Japanese
3. Change schedule using app, not settings.json
4. Restart app

Are you using the Microsoft Store version of WDD?: No

This is the very same issue I've had here #432 
So the app continues to write down time as region-localized string
Log contains:
```
[2022-10-10 06:40:09.947] System.FormatException: The string '午前 7:00:00' was not recognized as a valid DateTime. There is an unknown word starting at index '0'.
   at System.DateTime.Parse(String s, IFormatProvider provider)
   at WinDynamicDesktop.SunriseSunsetService.GetUserProvidedSolarData(DateTime date)
   at WinDynamicDesktop.SunriseSunsetService.GetSolarData(DateTime date)
   at WinDynamicDesktop.WallpaperEngine.RunScheduler(Boolean forceImageUpdate)
   at WinDynamicDesktop.AppContext..ctor(String[] args)
   at WinDynamicDesktop.Program.Main(String[] args)
```

**Comment by t1m0thyj** [comment]
@swoy45 Thanks for reporting this, could you try downloading the EXE from the latest build and let me know if it works?
https://github.com/t1m0thyj/WinDynamicDesktop/actions/runs/3389971915

**Comment by t1m0thyj** [comment]
This should be fixed in WDD 5.2.0, which was just published to GitHub and will be available on the Microsoft Store soon.
